src/ApplicationCore/Extensions/StringExtensions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Text.Json;
+
+namespace Microsoft.eShopWeb
+{
+    public static class StringExtensions
```

Comment:
Could we name this `JsonExtensions` instead?

Good call on pulling this out in to an Extensions class. I was looking through to see if we could set these settings globally so we don't have to have an Extensions class but it doesn't look like that is scheduled until .NET 5.